src/Map/types/viewerProperty.ts

```diff
@@ -0,0 +1,186 @@
+import type { LUT, Camera } from "../../mantle";
+
+export type SceneMode = "3d" | "2d" | "columbus";
+
+export type ViewerProperty = {
```

Comment:
Could you write a rule about how to add new property or how these properties are organized?

Reply:
That's great description. Thank you:pray: